src/modules/Projects/components/ProjectData/ProjectData.jsx

```diff
@@ -10,15 +11,19 @@ export default function ProjectData() {
     handleSubmit,
     formState: { errors },
   } = useForm();
+
   let onsubmit = async (data) => {
-    let response = await axios.post(`${baseurl}/Project`, data , {
+    let response = await axios.post(`${baseurl}/Project`, data, {
```

Comment:
We need to wrap  api call inside a try catch block to catch the error coming from the backend and display messages for that.